src/framework/preferences/js/Panels.js

```diff
@@ -916,10 +924,10 @@ var fluid_2_0 = fluid_2_0 || {};
         },
         selectors: {
             links: ".flc-prefsEditor-links",
-            linksChoiceLabel: ".flc-prefsEditor-links-choice-label"
+            linkschoiceLabel: ".flc-prefsEditor-links-choice-label"
```

Comment:
Was there a reason to change the name from "linksChoiceLabel" to "linkschoiceLabel"? Similar for "inputschoiceLabel".